src/components/RadioWithTextField.tsx

```diff
@@ -49,6 +56,18 @@ export function RadioWithTextField({
     }
   }, [openTextField]);
 
+  useEffect(() => {
+    if (fieldIsFromIPA) {
```

Comment:
Merge this conditions

Reply:
I can write a cleaner condition but i need to distinguish the fields that came from ipa from the ones who doesn't in order to set that type of logic